unit/deserialize_test.go

```diff
@@ -218,7 +219,7 @@ Description= words here `),
 Description= words here \
   `),
 			[]*UnitOption{
-				&UnitOption{"Unit", "Description", "words here"},
+				&UnitOption{"Unit", "Description", "words here \\"},
```

Comment:
For the `end_to_end_test.go`, you would add the following test:

```
{
			`[Unit]
Description=Demo \

Requires=docker.service
`,
			`[Unit]
Description=Demo \
Requires=docker.service
`,
		},
```

This tests is the current state, but this is IMHO broken. I would expect that it keeps the empty line (or drop the \ in this case).